.env

```diff
@@ -10,4 +10,5 @@
 
 # i.e. `npm run migrate`
 
-DATABASE_URL="postgresql://admin:postgrespw@localhost:5432/mydb?schema=public"
+# DATABASE_URL="postgresql://admin:postgrespw@localhost:5432/mydb?schema=public"
+DATABASE_URL="postgres://bklkfhcurrjrdg:45f24c413ea460cd7eaa71a0bf4de6363f477cbf1478973e87c0d9dc0e0f402e@ec2-34-225-159-178.compute-1.amazonaws.com:5432/dfjkm0mf8pomhq"
```

Comment:
This is terrible form but from what I have read/experimented with this is the only way. For some reason heroku can't read it's own `DATABASE_URL` config var in the app so deployment always fails if I don't use the `DATABASE_URL` from the `.env` file. I experienced this at work earlier this year as well and I am not aware of a fix. `prisma.schema` Given this is a private repo I think we're okay for this project but if there are security concerns let me know.

Reply:
Reference: https://stackoverflow.com/questions/61639148/heroku-database-url-is-undefined-in-nodejs-app

Reply:
I'm pretty sure you can store environment variables on heroku https://devcenter.heroku.com/articles/config-vars

Our problem is probably because `.env` isn't in `.gitignore` so Heroku is reading from here instead of it's config variables. Instead, let's just add `.env` to `.gitignore` since everything stored in here atm is only for local use, and hopefully that fixes the problem

Reply:
@keithradford My bad for not mentioning it, that was one of the things I did try before this PR with no luck. See attached, this is when I supply no `.env` file 
<img width="618" alt="image" src="https://user-images.githubusercontent.com/31942169/175614002-387d57fb-39b0-4859-b686-7dab6d9438b3.png">

The variable is set in the config vars on Heroku (checked in the UI) so its definitely there

Reply:
Sounds like we shouldn't be hardcoding it as the database URL is subject to change on certain conditions outside of our control.

Does this help? https://devcenter.heroku.com/articles/connecting-to-heroku-postgres-databases-from-outside-of-heroku#don-t-copy-and-paste-credentials-to-a-separate-environment-or-app-code

Sounds like we can run the app like `DATABASE_URL=$(heroku config:get DATABASE_URL -a your-app) your_process`